ecocode-rules-specifications/pom.xml

```diff
@@ -0,0 +1,178 @@
+			<artifactId>junit-jupiter</artifactId>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.jacoco</groupId>
+				<artifactId>jacoco-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>prepare-agent</id>
+						<goals>
+							<goal>prepare-agent</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>report</id>
+						<goals>
+							<goal>report</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.asciidoctor</groupId>
+				<artifactId>asciidoctor-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>convert-to-html</id>
+						<phase>generate-resources</phase>
+						<goals>
+							<goal>process-asciidoc</goal>
+						</goals>
+						<configuration>
+							<sourceDirectory>${project.basedir}/src/main/rules</sourceDirectory>
+							<outputDirectory>${project.build.directory}/rules</outputDirectory>
+							<attributes>
+								<source-highlighter>coderay</source-highlighter>
+								<coderay-css>style</coderay-css>
+							</attributes>
+							<preserveDirectories>true</preserveDirectories>
+							<headerFooter>false</headerFooter>
+							<relativeBaseDir>true</relativeBaseDir>
+							<logHandler>
+								<failIf>
+									<severity>ERROR</severity>
+								</failIf>
+							</logHandler>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-antrun-plugin</artifactId>
+				<version>3.1.0</version>
+				<executions>
+					<execution>
+						<phase>process-resources</phase>
+						<configuration>
+							<target>
+								<copy todir="${project.build.outputDirectory}/io/ecocode/rules/java">
+									<fileset dir="${project.build.directory}/rules">
+										<include name="*/java/EC*.html"/>
+										<include name="*/EC*.json"/>
+									</fileset>
+									<flattenmapper/>
+								</copy>
+								<copy todir="${project.build.outputDirectory}/io/ecocode/rules/php">
+									<fileset dir="${project.build.directory}/rules">
+										<include name="*/php/EC*.html"/>
+										<include name="*/EC*.json"/>
+									</fileset>
+									<flattenmapper/>
+								</copy>
+								<copy todir="${project.build.outputDirectory}/io/ecocode/rules/python">
+									<fileset dir="${project.build.directory}/rules">
+										<include name="*/python/EC*.html"/>
+										<include name="*/EC*.json"/>
+									</fileset>
+									<flattenmapper/>
+								</copy>
+							</target>
+						</configuration>
+						<goals>
+							<goal>run</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
```

Comment:
add here the version of plugin (and not in parent pom) becsaue only used in this module